policyengine_canada/parameters/gov/provinces/nl/tax/income/credits/physical_activity_tax_credit/age_eligible.yaml

```diff
@@ -0,0 +1,9 @@
+description: Newfoundland child age eligiblity for the Physical activity tax credit.
```

Comment:
```suggestion
description: Newfoundland limits its Physical Activity Tax Credit to people below this age.
```

Reply:
Capitalize the words in the program everywhere else please.

Reply:
add the reference: https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/newfoundland-labrador/5001-pc/information-residents-newfoundland-labrador.html#PATC_NL